addroom.html

```diff
@@ -132,12 +141,13 @@
 
   <body>
       <ul>
-          <li><img  src="./img/logo.png" width="40%" height="40%"><li>
-          <li><a href="/homepage.html">Home</a></li>
-          <li><a href="/signup.html">Sign Up</a></li>
-          <li><a href="/signin.html">Sign In</a></li>
-          <li><a href="/">Sign Out</a></li>
-          <li><a href="/">About</a></li>
+          <li><img  src="./img/logo.png" width="10%" height="10%"></li>
```

Comment:
We can replace the navigation bar with something like this: http://bit.ly/2T66RYj . Hard-coded left margin would better not be used.

Reply:
OK. I will try it later